### PR TITLE
Ret 2058

### PR DESCRIPTION
**Pull request by robin-ieong**
(https://tools.hmcts.net/jira/browse/RET-2058)

### Change description ###
A Respondent has submitted an ET3 Form which has been vetted and accepted by case workers. Now all parties need to be told the form has been received and accepted, and a copy of the ET3 form must be sent to the Claimant.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: definitions/json/CaseEventToFields.json [hunk 1]
```diff
@@ -4913,6 +4913,170 @@
...
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "et3Notification",
+    "CaseFieldID": "et3SendDocByFirstClass",
+    "DisplayContext": "READONLY",
+    "PageID": 3,
+    "PageDisplayOrder": 3,
+    "PageFieldDisplayOrder": 2,
+    "PageLabel": "Send documents",
```
Comment (jsinghm13) [inline definitions/json/CaseEventToFields.json:5004]: I don't think you need to set this value again when it has been already set in the first field on the page

**Review by jsinghm13 — APPROVED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -11981,6 +12011,426 @@
...
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "et3SelectAllThatApply",
+    "UserRole": "caseworker-employment-etjudge",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "et3OtherTypeDocumentName",
+    "UserRole": "caseworker-employment",
```
Comment (Itay-Kirsh) [inline definitions/json/AuthorisationCaseField.json:12077]: R only role

File: definitions/json/AuthorisationCaseField.json [hunk 3]
```diff
@@ -11981,6 +12011,426 @@
...
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "et3OtherTypeDocumentName",
+    "UserRole": "caseworker-employment-etjudge",
```
Comment (Itay-Kirsh) [inline definitions/json/AuthorisationCaseField.json:12083]: R only role

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 4]
```diff
@@ -11981,6 +12011,426 @@
...
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "et3ClaimantAndRespondentAddresses",
+    "UserRole": "caseworker-employment-englandwales",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "et3ClaimantAndRespondentAddresses",
+    "UserRole": "caseworker-employment",
+    "CRUD": "CRUD"
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:12246]: R

File: definitions/json/CaseEventToFields.json [hunk 5]
```diff
@@ -3526,7 +3526,6 @@
     "PageID": 3,
     "PageDisplayOrder": 3,
     "PageFieldDisplayOrder": 2,
-    "PageLabel": "Send documents",
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields.json:3529]: Intentional?

File: definitions/json/CaseEventToFields.json [hunk 6]
```diff
@@ -5033,6 +5032,170 @@
     "RetainHiddenValue": "Yes",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "et3Notification",
+    "CaseFieldID": "et3NotificationDocCollection",
+    "DisplayContext": "MANDATORY",
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields.json:5039]: This should be COMPLEX - the fields part of the complexType should then be listed on EventToComplexTypes

**Review by jsinghm13 — DISMISSED** [review]

**Review by Itay-Kirsh — APPROVED** [review]

**Review by jsinghm13 — APPROVED** [review]

**Pull request merged**